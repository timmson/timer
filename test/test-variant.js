import React from "react"
import renderer from "react-test-renderer"

import Variant from "../src/variant"

describe("Variant should", () => {

	test("return variant", () => {
		const component = renderer.create(<Variant id={1} value={"01:00"}/>)
		expect(component.toJSON()).toMatchSnapshot()
		component.unmount()
	});

	test("contain properties", () => {
		expect(Variant.propTypes).toHaveProperty("value")
	});
});