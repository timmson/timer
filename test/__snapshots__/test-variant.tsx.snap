// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Variant should return variant 1`] = `
<div
  className="variants col-sm"
>
  <a
    href="#"
    onClick={[Function]}
  >
    01:00
  </a>
</div>
`;
